.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+  - "node"
+  - "6"
+  - "4"
```

Comment:
Makes sense to add .editorconfig and ensure newlines in every repo as we touch them.